OpenSim/Simulation/Wrap/WrapSphereObst.cpp

```diff
@@ -57,46 +53,17 @@ WrapSphereObst::~WrapSphereObst()
 {
 }
 
-//_____________________________________________________________________________
-/**
-* Copy constructor.
-*
-* @param aWrapSphereObst WrapSphereObst to be copied.
-*/
-WrapSphereObst::WrapSphereObst(const WrapSphereObst& aWrapSphereObst) :
-WrapObject(aWrapSphereObst),
-_radius(_radiusProp.getValueDbl()),
-_length(_lengthProp.getValueDbl())
-{
-    setNull();
-    setupProperties();
-    copyData(aWrapSphereObst);
-}
-
 //=============================================================================
 // CONSTRUCTION METHODS
 //=============================================================================
-//_____________________________________________________________________________
-/**
-* Set the data members of this WrapSphere to their null values.
-*/
-void WrapSphereObst::setNull()
-{
-}
-
 //_____________________________________________________________________________
 /**
 * Connect properties to local pointers.
 */
-void WrapSphereObst::setupProperties()
+void WrapSphereObst::constructProperties()
 {
-    _radiusProp.setName("radius");
-    _radiusProp.setValue(-1.0);
-    _propertySet.append(&_radiusProp);
-
-    _lengthProp.setName("length");
-    _lengthProp.setValue(1.0);
-    _propertySet.append(&_lengthProp);
+    constructProperty_radius(1.0);
+    constructProperty_length(1.0);
```

Comment:
Why does a sphere have a `length` property?

Reply:
I have no clue... I am not even sure of the difference between WrapSphere and WrapSphereObst. I removed the length property and I'll insert it back if someone says it make sense to have it!

Reply:
Actually, it seems to refers to a "height" as shown in getDimensionsString. This may be due to a bad choice of wording in the past. I think I changed my mind and I'll leave it there until someone else confirm I should remove it